fvm/evm/stdlib/contract.go

```diff
@@ -49,6 +49,7 @@ const evmAddressTypeQualifiedIdentifier = "EVM.EVMAddress"
 const evmBalanceTypeQualifiedIdentifier = "EVM.Balance"
 const evmResultTypeQualifiedIdentifier = "EVM.Result"
 const evmStatusTypeQualifiedIdentifier = "EVM.Status"
+const evmBlockTypeQualifiedIdentifier = "EVM.EVMBlock"
```

Comment:
```suggestion
const evmBlockTypeQualifiedIdentifier = "EVM.Block"
```

Reply:
That unfortunately is not possible:

```bash
error: cannot redeclare structure: `Block` is already declared
        	            	   --> 0000000000000001.EVM:468:11
        	            	    |
        	            	468 |     struct Block {
```

Reply:
`Block` is already taken, because it is defined by Cadence `stdlib`. The same goes for `Address`, that's why the struct here is named `EVMAddress`.